app/mailers/sympa_subscribe_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class SympaSubscribeMailer < ApplicationMailer
+
+    def register_for_announce_list
+
+        unless $SUBSCRIBE_LIST_NAME.nil? || $SUBSCRIBE_LIST_NAME.empty?
```

Comment:
1. Use the existent `$ANNOUNCE_LIST` variable name
2. Change the `SUBSCRIBE_SERVICE_MAIL` name to `ANNOUNCE_SERVICE_HOST`